tools/windows/install-help/cal/customaction.vcxproj

```diff
@@ -172,7 +172,7 @@
     <ClCompile Include="PropertyReplacer.cpp" />
     <ClCompile Include="rollback.cpp" />
     <ClCompile Include="TargetMachine.cpp" />
-    <ClCompile Include="stdafx.cpp">
+    <ClCompile Include="precompiled\stdafx.cpp">
```

Comment:
I see may changes to direct PCH to a different directory. Why? Did you have some sort of conflict?